helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -277,10 +277,13 @@ k8sattributes/metrics:
   pod_association:
     - sources:
       - from: resource_attribute
-        name: k8s.node.name
+        name: k8s.pod.uid
     - sources:
       - from: resource_attribute
-        name: k8s.pod.uid
+        name: k8s.namespace.name
```

Comment:
I don't think we should have namespace and node by themself in any of our k8sattributes processors. Just by themself, the processor can associate any pod which has same namespace or node.

These should have a 2nd condition like the `k8s.pod.name` + `k8s.node.name`.

Is below still relevant? I have not seen a resource_attr `ip` so wonder if this is some old attr replaced by k8s.pod.ip?
```
    - sources:
      - from: resource_attribute
        name: ip
 ```

I believe we'll have to make similarly fixes for the k8sattributes processor in logs pipeline.

Reply:
I would suggest to leave it in. It is easy for an app to set it with the k8s downward api. Can you confirm it is usable in pod_association from the k8sattributes docs also?